script/remove_empty_cached_counts_related_to_overviews.rb

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+
+require File.dirname(__FILE__) + '/../config/environment'
+Card::Auth.current_id = Card::WagnBotID
+
+[Card::WikirateAnalysisID, Card::WikirateTopicID, Card::WikirateCompanyID].each do |type|
+  cc_ids = Card.search(
+          right: { codename: 'cached_count'},
```

Comment:
Indent the first parameter one step more than the start of the previous line.<br>Space inside } missing.